BossMod/BossModule/AIHints.cs

```diff
@@ -86,9 +86,19 @@ public void FillPotentialTargets(WorldState ws, bool playerIsDefaultTank)
     {
         bool playerInFate = ws.Client.ActiveFate.ID != 0 && ws.Party.Player()?.Level <= Service.LuminaRow<Lumina.Excel.GeneratedSheets.Fate>(ws.Client.ActiveFate.ID)?.ClassJobLevelMax;
         var allowedFateID = playerInFate ? ws.Client.ActiveFate.ID : 0;
-        foreach (var actor in ws.Actors.Where(a => a.Type == ActorType.Enemy && a.IsTargetable && !a.IsAlly && !a.IsDead && (a.FateID == 0 || a.FateID == allowedFateID)))
+        foreach (var actor in ws.Actors.Where(a => a.Type == ActorType.Enemy && a.IsTargetable && !a.IsAlly && !a.IsDead))
         {
-            PotentialTargets.Add(new(actor, playerIsDefaultTank && (actor.InCombat || actor.FateID != 0)));
+            // fate mob in fate we are NOT a part of, skip entirely. it's okay to "attack" these (i.e., they won't be added as forbidden targets) because we can't even hit them
+            // (though aggro'd mobs will continue attacking us after we unsync, but who really cares)
+            if (actor.FateID > 0 && actor.FateID != allowedFateID)
+                continue;
+
+            var allowedAttack = actor.InCombat && ws.Party.FindSlot(actor.TargetID) >= 0;
```

Comment:
So uhh, if i pull a mob and no one targets it, i'm fucked? I don't really follow, why is it needed?

Reply:
if you pull a mob and no one targets it, then it would be targeting you, so nothing would really change. this is just to prevent ai mode from attacking random other players' targets in overworld

Reply:
Ah right i see, ok